parallel-consumer-core/src/main/java/io/confluent/parallelconsumer/internal/ProducerManager.java

```diff
@@ -114,15 +113,15 @@ private TransactionManager getTransactionManager() {
     }
 
     /**
-     * Produce a message back to the broker.
-     * <p>
-     * Implementation uses the blocking API, performance upgrade in later versions, is not an issue for the more common
-     * use case where messages aren't produced.
+     * Produce messages back to the broker.
+     *
+     * <p>Produced messages are not waited for, instead their futures are returned so the caller can
```

Comment:
was this you, or the formatter that moved the <p>? Rest of the project has <p> on the blank line